Gems/ROS2/Code/Source/SdfAssetBuilder/SdfAssetBuilderSettings.cpp

```diff
@@ -60,13 +61,37 @@ namespace ROS2
                 ->Version(0)
                 ->Field("UseArticulations", &SdfAssetBuilderSettings::m_useArticulations)
                 ->Field("URDFPreserveFixedJoint", &SdfAssetBuilderSettings::m_urdfPreserveFixedJoints)
+                ->Field("ImportReferencedMeshFiles", &SdfAssetBuilderSettings::m_importReferencedMeshFiles)
```

Comment:
To discuss only! It need some explanation in o3de.org and  link here. Let's do that in ext steps.

Reply:
Yeah we need to document these settings on docs.o3de.org